### DISCUSSION TITLE
MACE models for chloride containing molecules

### DISCUSSION THREAD

**Discussion by kuntalg97** [ACEsuit/mace#1147] [Q&A] [answered]
Hi,

I have a system consisting of carbon tetrachloride, methyl chloride, and ethyl hypochlorite. I plan to perform reactive simulations in LAMMPS with these molecules under non-polar conditions and I am considering using MACE-OFF for this purpose. Preliminary tests indicated that MACE-OFF works quite well for bulk carbon tetrachloride, especially in capturing structural correlations. However, methyl chloride and ethyl hypochlorite tend to dissociate non-physically after a certain period.

Is there a way to resolve this issue without having to fine-tune MACE-OFF? Alternatively, is there another MACE model that would be better suited for this type of system?

Thank you!

**Comment by ilyes319** [comment, marked as answer]
Hey,

You could try finetuning by sampling some configurations and re evaluate with DFT. You could also give a try to the mace-omol model we just release, it has much better reactivity than mace-off.

You can see here how to use it: https://github.com/ACEsuit/mace/releases/tag/v0.3.14

**Reply by kuntalg97** [reply]
Thanks!

Has anyone used MACE-OMOL or any other MACE models to study chemical reactions involving any of the molecules (or similar) I mentioned above? I couldn’t find any such examples.

**Reply by ilyes319** [reply]
Nothing published to my knowledge but i ve seen good results on similar systems.

**Reply by kuntalg97** [reply]
Okay thanks! We'll try it out.

**Reply by Swiftily** [reply]
> Nothing published to my knowledge but i ve seen good results on similar systems.

If we don’t take computational cost into account, does the MACE-OMOL model generally achieve much higher accuracy than MACE-OFF?
I've noticed that MACE-OFF (both ’23 and ’24 versions) performs poorly in frequency calculations for certain systems.

**Reply by ilyes319** [reply]
yes it should be generally more accurate.
